config/taxonomy/example/application/appinfo.yaml

```diff
@@ -0,0 +1,21 @@
+definitions:
+  AppInfo:
+    type: object
```

Comment:
Type should be defined in the base, here we should have only properties.

Reply:
Should AppInfo be a part of the base.yaml. Asking this because you are referring to appinfo in taxonomy.json and hence it should it not include appinfo ?

Reply:
I just added AppInfo to base.yaml